user/api_serializers.py

```diff
@@ -0,0 +1,25 @@
+"""
+	Serializers defined for rest framework responses
+"""
+
+from rest_framework import serializers
+from .models import Post
+
+class RecentPostSerializer(serializers.HyperlinkedModelSerializer):
+	"""
+		Serializes a limited subsection of fields from the Post model.
+		Intended for use by the Recent Activity endpoint.
+	"""
+	user = serializers.PrimaryKeyRelatedField(read_only=True, default=serializers.CurrentUserDefault())
+	class Meta:
+		model = Post
+		fields = ['interacted', 'updated', 'title', 'id', 'url', 'comments_count', 'user']
```

Comment:
Does this inherit our permissions model somewhere further in or should we be addressing private/members only blogs somewhere in here?

Reply:
From the behavior I'm seeing from testing, it correctly inherits permissions - I'll have to triple check though that it isn't just doing magic filtering on the client side.

Reply:
Yeah, it's following the same logic (And permissions) as `PostSerializer`. No client-side filtering - the permissions seem to be catered for in the Post Model (`models.py:129`).